admin/src/components/ContributionLinkForm.vue

```diff
@@ -157,23 +164,33 @@ export default {
       if (this.form.validFrom === null)
         return this.toastError(this.$t('contributionLink.noStartDate'))
       if (this.form.validTo === null) return this.toastError(this.$t('contributionLink.noEndDate'))
+
+      const { validFrom, validTo, name, amount, memo, cycle, maxPerCycle, maxAmountPerMonth } =
+        this.form
+      const variables = {
+        validFrom,
+        validTo,
+        name,
+        amount,
+        memo,
+        cycle,
+        maxPerCycle,
+        maxAmountPerMonth,
+        id: this.contributionLinkData.id ? this.contributionLinkData.id : null,
+      }
+
       this.$apollo
         .mutate({
-          mutation: createContributionLink,
-          variables: {
-            validFrom: this.form.validFrom,
-            validTo: this.form.validTo,
-            name: this.form.name,
-            amount: this.form.amount,
-            memo: this.form.memo,
-            cycle: this.form.cycle,
-            maxPerCycle: this.form.maxPerCycle,
-            maxAmountPerMonth: this.form.maxAmountPerMonth,
-          },
+          mutation: this.editContributionLink ? updateContributionLink : createContributionLink,
+          variables: variables,
         })
         .then((result) => {
-          this.link = result.data.createContributionLink.link
-          this.toastSuccess(this.link)
+          this.link = this.editContributionLink
```

Comment:
Why `this`? There is no data for link.
`const link = .... `

Reply:
changed to const